railties/lib/rails/commands/destroy/destroy_command.rb

```diff
@@ -22,6 +27,7 @@ def perform(*)
         load_generators
 
         Rails::Generators.invoke generator, args, behavior: :revoke, destination_root: Rails::Command.root
+        return delete_css_file_generate_with_scaffold if generator == "scaffold"
```

Comment:
@AdamMusa I don't see any tests for this command right now, would you be interested in adding them in this PR? It might help getting this merged sooner. 🙏

Reply:
asap

Reply:
please @zzak  where would be the ideal place to place the test of my helper function?
Or I can create a test file and write my test

Reply:
@zzak I found a suitable place to write the test (railties/test/generators/scaffold_generator_test.rb in this file). But I will be delighted if you suggest me a place to write.

Reply:
@AdamMusa That seems like a good place to start 🙏

Reply:
this good the test is already done